docs/install.md

```diff
@@ -1,26 +1,24 @@
-# Service and Logging 
+# Running k0s as a service
 
-We've created the subcommand `k0s install` to allow users to easily install k0s as a service, and define its logging.
-
-This is an **alpha state** feature.
+Sub-command `k0s install` allows users to easily install k0s as a service, and define its logging.
 
 ## Caveats
 * This command is strictly a helper command. It is not meant to provide a fully-automated solution, since you can run k0s in multiple, very different ways.
 * It configures your service set-up as either a worker or a server, and will have different tasks, depending on the role you pick.
 * Supported services: OpenRC & Systemd
 
-### Server setup
```

Comment:
Perhaps adding an example here how to setup server node with install command
```
k0s install server --config cfg.yaml
```
and similar more details can be found here https://github.com/k0sproject/k0s/pull/594#issuecomment-754003983

Reply:
This could be done with a separate PR. Probably a similar example should be provided for the worker setup?

Reply:
Ok then we should reference this in an issue so we don't miss it.